helloworld.js

```diff
@@ -0,0 +1,2 @@
+
+console.log('hello, Ron Oh');
```

Comment:
I See you have made two pull requests, If you decide to push a different branch up as your assignment, you should close the pull request you are not submitting
 :)